bindings/python/src/OpenSpaceToolkitPhysicsPy/Units/Derived/Angle.cpp

```diff
@@ -77,70 +78,69 @@ inline void OpenSpaceToolkitPhysicsPy_Units_Derived_Angle(pybind11::module& aMod
-            {
-                return anAngle.inDegrees();
-            }
+            overload_cast<const Real&, const Real&>(&Angle::inDegrees, const_),
+            arg("lower_bound"),
+            arg("upper_bound")
         )
+        .def("in_arcminutes", overload_cast<>(&Angle::inArcminutes, const_))
         .def(
             "in_arcminutes",
-            +[](const Angle& anAngle) -> Real
-            {
-                return anAngle.inArcminutes();
-            }
+            overload_cast<const Real&, const Real&>(&Angle::inArcminutes, const_),
+            arg("lower_bound"),
+            arg("upper_bound")
         )
+        .def("in_arcseconds", overload_cast<>(&Angle::inArcseconds, const_))
         .def(
             "in_arcseconds",
-            +[](const Angle& anAngle) -> Real
-            {
-                return anAngle.inArcseconds();
-            }
+            overload_cast<const Real&, const Real&>(&Angle::inArcseconds, const_),
+            arg("lower_bound"),
+            arg("upper_bound")
         )
         .def("in_revolutions", &Angle::inRevolutions)
-        .def("to_string", &Angle::toString, "aPrecision"_a = Integer::Undefined())
+        .def("to_string", &Angle::toString, arg("do_sanitize") = false)
 
+        // Define static methods
         .def_static("undefined", &Angle::Undefined)
         .def_static("zero", &Angle::Zero)
         .def_static("half_pi", &Angle::HalfPi)
         .def_static("pi", &Angle::Pi)
         .def_static("two_pi", &Angle::TwoPi)
-        .def_static("radians", &Angle::Radians)
-        .def_static("degrees", &Angle::Degrees)
-        .def_static("arcminutes", &Angle::Arcminutes)
-        .def_static("arcseconds", &Angle::Arcseconds)
-        .def_static("revolutions", &Angle::Revolutions)
-        .def(
-            "between_vector2d",
-            +[](const Vector2d& aFirstVector, const Vector2d& aSecondVector) -> Angle
-            {
-                return Angle::Between(aFirstVector, aSecondVector);
-            }
+        .def_static("radians", &Angle::Radians, arg("value"))
+        .def_static("degrees", &Angle::Degrees, arg("value"))
+        .def_static("arcminutes", &Angle::Arcminutes, arg("value"))
+        .def_static("arcseconds", &Angle::Arcseconds, arg("value"))
+        .def_static("revolutions", &Angle::Revolutions, arg("value"))
+        .def_static(
+            "between",
+            overload_cast<const Vector2d&, const Vector2d&>(&Angle::Between),
+            arg("first_vector"),
+            arg("second_vector")
         )
-        .def(
-            "between_vector3d",
-            +[](const Vector3d& aFirstVector, const Vector3d& aSecondVector) -> Angle
-            {
-                return Angle::Between(aFirstVector, aSecondVector);
-            }
+        .def_static(
+            "between",
+            overload_cast<const Vector3d&, const Vector3d&>(&Angle::Between),
+            arg("first_vector"),
+            arg("second_vector")
```

Comment:
agreed that this is cleaner to have overload in python reflect the overload in cpp but this is breaking potentially in some notebooks, just fyi to update the rest accordingly in parallel

Reply:
Nice!!

Reply:
i checked and we should be good with the change